api/src/main/java/com/google/sps/GetUsername.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.Firestore;
+import com.google.firebase.cloud.FirestoreClient;
+import com.google.api.core.ApiFuture;
+
+@RestController
+public class GetUsername {
+  @PostMapping("/getUsername")
+  String getUsername(@RequestBody String uid) throws Exception {
+    Firebase.init();
```

Comment:
#58 fixes the need for this line. For now, this is a no-op, but after that is merged we should remove this line

Reply:
Good to know, should I change this now or wait until #58 is merged?

Reply:
Hmm I want to say that `master` branch should always contain as little broken code as possible. So I feel like we should wait, and just use this comment thread as a reminder to fix sometime. Though really it's such a small thing and both branches will probably be merged so soon that the difference is negligible..

Reply:
Technically this code is not broken since it works fine, it's just not ideal/best practice. We can wait to merge this until #58 is handled though.